cranelift/isle/isle/src/sema.rs

```diff
@@ -1113,18 +1116,41 @@ impl TypeEnv {
 #[derive(Clone, Debug, Default)]
 struct Bindings {
     seen: Vec<BoundVar>,
+    next_scope: usize,
+    in_scope: Vec<usize>,
```

Comment:
Nit: maybe a comment for whether the indices on this vec are meaningful?

Reply:
Sure, more documentation is rarely a bad idea. :grin: Thanks!